server/src/main/java/com/datastrato/gravitino/server/GravitinoServer.java

```diff
@@ -113,6 +109,16 @@ protected void configure() {
     server.addCustomFilters(API_ANY_PATH);
     server.addFilter(new VersioningFilter(), API_ANY_PATH);
     server.addSystemFilters(API_ANY_PATH);
+
+    boolean enableAuthorization = serverConfig.get(Configs.ENABLE_AUTHORIZATION);
+    if (enableAuthorization) {
+      List<String> accessControlPaths =
+          Lists.newArrayList("/api/users", "/api/groups", "/api/admins");
```

Comment:
The path is not correct, it should `api/metalakes/{metalake}/users`, right?

Reply:
Yes, I will correct the paths and test it.

Reply:
Done.